src/components/Datagrid/customCells/NumberCell.tsx

```diff
@@ -21,8 +21,7 @@ export interface NumberCellProps {
 
 export type NumberCell = CustomCell<NumberCellProps>;
 
-const onlyDigitsRegExp = /^\d+$/;
-const flaotingPointDigits = /^[0-9]+[.,]?[0-9]+$/;
+const floatOrDigits = /^\d+$|^[0-9]+[.,]?[0-9]+$/;
```

Comment:
It's basically an or for this change - https://github.com/saleor/saleor-dashboard/pull/4335/files#diff-a2aaa4b69376c122004cd558a0107a44ec7023468283292f3dd69ee4ab5de281R85-R88